neocore/Cryptography/MerkleTree.py

```diff
@@ -156,7 +156,7 @@ def Trim(self, flags):
         Args:
             flags: "0000" for trimming, any other value for keeping the nodes.
         """
-        logger.info("Trimming!")
+        print("Trimming!")
```

Comment:
this can be removed completely. Otherwise we'll get a lot of "Trimming" messages for each block that gets recalculated.

Reply:
I figured you would say that but I was trying to translate as close to 1:1 on the first pass.